dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -384,33 +392,29 @@ protected void githubWebhookRelease(String repository, String username, String g
-            isSuccessful &= createWorkflowsAndVersionsFromDockstoreYml(dockstoreYaml12.getTools(), repository, gitReference, installationId, username, dockstoreYml, AppTool.class, messageWriter);
-            isSuccessful &= createWorkflowsAndVersionsFromDockstoreYml(dockstoreYaml12.getNotebooks(), repository, gitReference, installationId, username, dockstoreYml, Notebook.class, messageWriter);
+            isSuccessful &= createWorkflowsAndVersionsFromDockstoreYml(services, repository, gitReference, installationId, username, dockstoreYml, Service.class, lambdaEventGroupId);
+            isSuccessful &= createWorkflowsAndVersionsFromDockstoreYml(dockstoreYaml12.getWorkflows(), repository, gitReference, installationId, username, dockstoreYml, BioWorkflow.class, lambdaEventGroupId);
+            isSuccessful &= createWorkflowsAndVersionsFromDockstoreYml(dockstoreYaml12.getTools(), repository, gitReference, installationId, username, dockstoreYml, AppTool.class, lambdaEventGroupId);
+            isSuccessful &= createWorkflowsAndVersionsFromDockstoreYml(dockstoreYaml12.getNotebooks(), repository, gitReference, installationId, username, dockstoreYml, Notebook.class, lambdaEventGroupId);
 
         } catch (Exception ex) {
 
             // If an exception propagates to here, log something helpful and abort .dockstore.yml processing.
             isSuccessful = false;
-            String msg = "User " + username + ": Error handling push event for repository " + repository + " and reference " + gitReference + "\n" + generateMessageFromException(ex);
-            LOG.info(msg, ex);
-            messageWriter.println(msg);
-            messageWriter.println("Terminated processing of .dockstore.yml.");
-
-            throw new CustomWebApplicationException(msg, statusCodeForLambda(ex));
-
-        } finally {
+            String msg = "Error handling push event for repository " + repository + " and reference " + gitReference + ":\n- " + generateMessageFromException(ex) + "\nTerminated processing of .dockstore.yml";
+            LOG.info("User " + username + ": " + msg, ex);
 
             // Make an entry in the github apps logs.
-            final boolean finalIsSuccessful = isSuccessful;
             new TransactionHelper(sessionFactory).transaction(() -> {
-                LambdaEvent lambdaEvent = createBasicEvent(repository, gitReference, username, LambdaEvent.LambdaEventType.PUSH);
-                lambdaEvent.setSuccess(finalIsSuccessful);
-                setEventMessage(lambdaEvent, stringWriter.toString());
+                LambdaEvent lambdaEvent = createBasicEvent(repository, gitReference, username, LambdaEvent.LambdaEventType.PUSH, false, lambdaEventGroupId);
+                setEventMessage(lambdaEvent, msg);
                 lambdaEventDAO.create(lambdaEvent);
             });
 
+            if (throwIfNotSuccessful) {
+                throw new CustomWebApplicationException(msg, statusCodeForLambda(ex));
+            }
```

Comment:
@denis-yuen This is the fix to prevent the GitHub installation from throwing an exception. We forgot to check the throwIfNotSuccessful boolean before throwing